ui/dialog.js

```diff
@@ -155,6 +155,7 @@ return $.widget( "ui.dialog", {
 		var next,
 			originalPosition = this.originalPosition;
 
+		this._stopTrackingInstance();
```

Comment:
He didn't mean rename it, he meant there's already a function that does this, and it's called `_untrackInstance()`.